src/containers/Bookings.js

```diff
@@ -1,21 +1,64 @@
+
 import React, { Component } from "react";
-import Search from "../components/Search.js";
-// import Results from "../components/Results.js";
-// import * as FakeBookings from "../data/fakeBookings.json";
+import Search from "../components/search/Search.js";
+import Results from "../components/results/Results.js";
+import * as FakeBookings from "../data/fakeBookings.json";
 
 export default class Bookings extends Component {
-  search = () => {
-    console.info("to do!");
-  };
-  
+  constructor(props) {
+    super(props);
+    this.state = {
+      FakeBookings: '',
+      term: '',
+      result: false,
+      direction: {
+        sortin: 'asc'
+      },
+      text: ''
+    }
+    this.searchHandler = this.searchHandler.bind(this)
```

Comment:
You can avoid having to do this (somewhat ugly) `.bind` by changing your methods to use the class properties syntax:

```js
searchHandler = (inputType) => {
  ...
}
```